### DISCUSSION TITLE
Parsing a MISP JSON file

### DISCUSSION THREAD

**Discussion by lattera** [MISP/PyMISP#836] [Q&A] [answered]
Let's say I've downloaded a MISP JSON file from a web server directly (for example, the JSON files from `https://www.circl.lu/doc/misp/feed-osint/`), could I have PyMISP parse that file? The documentation I'm seeing focuses on API access to a MISP server.

**Comment by Rafiot** [comment, marked as answer]
yes, you can use this method: https://pymisp.readthedocs.io/en/latest/modules.html#pymisp.MISPEvent.load_file

**Reply by lattera** [reply]
Awesome. Thanks!
